### PR TITLE
Make small changes to the story. Added an "end" folder

### PR DESCRIPTION
**Pull request by HuyenNguyen2302**
Please review this, @casey-collab

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/1535)

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, you there?

File: english/end/end.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Lol this is the dead-end page that I talked to you about ;)
+
+But actually this is just a joke. I hope you enjoyed the game :)
+
+Here is a couple of options that you will wanna pick:
+
+[If you wanna eat the marshmallow, please click here lol](../eatmarshmallow/eatmarshmallow.md)
+[If you want the DreamRide, please click here](../enjoy-DreamRide/enjoy-DreamRide.md)
```
Comment (casey-collab) [inline english/end/end.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

Comment (casey-collab) [inline english/end/end.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

File: english/end/end.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Lol this is the dead-end page that I talked to you about ;)
```
Comment (casey-collab) [inline english/end/end.md]: It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**